.claude/hooks/session-start.sh

```
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# SessionStart hook — smart welcome banner (new vs. existing project).
+#
+# NOTE: Phase 2 (STO-10) will upgrade this to a "count, don't claim" banner that
+# reports real scene/page counts. For now it ports the existing detection logic
+# into a maintainable script (was previously an inline command that never ran).
+
+if ls *.fountain >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)